lib/appConfig.ts

```diff
@@ -145,14 +152,22 @@ export const createAppConfig = (entries: { [entryAlias: string]: string }, optio
 				plugins.push(CSSEntryPointsPlugin({ createEmptyEntryPoints: options.createEmptyCSSEntryPoints }))
 			}
 
+			if (options.extractLicenseInformation) {
+				plugins.push(REUSELicensesPlugin(
+					typeof options.extractLicenseInformation === 'object'
+						? options.extractLicenseInformation
+						: {},
+				))
+			}
```

Comment:
Should we enable it only in production build?

Reply:
Sure makes sense!